sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -682,3 +683,43 @@ def test_y_std_with_multitarget_normalized():
     assert y_pred.shape == (n_samples, n_targets)
     assert y_std.shape == (n_samples, n_targets)
     assert y_cov.shape == (n_samples, n_samples, n_targets)
+
+
+def test_y_std_cov_with_multitarget():
```

Comment:
For this test, it would be better to use `pytest.mark.parametrize` on the previous test to avoid code redundancy.

In addition, I see that I already did a PR but forgot about it: https://github.com/scikit-learn/scikit-learn/pull/21996

It was covering one of the problems. Could you check that the solution is equivalent? I also see that we made an additional test to check the value of the std. dev. and covariance. This should be added in yours as well.

Reply:
Hi! Yes your solution in #21996 is equivalent. In addition, I added line 399-400 which make sure that `y_mean` is also the right shape corresponding to the docstring for `gpr.predict`.

I merged your fixes in with this branch and updated the test use `pytest.mark.parametrize` as suggested. Is the test for the values of std. dev. and covariance that you mentioned the one included in #21996?